src/main/java/AutoLotto/model/LottoMachine.java

```diff
@@ -6,33 +6,48 @@
 import java.util.List;
 import java.util.Set;
 
-import static autoLotto.model.LottoConstants.INVALID_PURCHASE_AMOUNT;
-
 public class LottoMachine {
-    private static final Long VALID_UNIT = 1000L;
+    private static final String INVALID_PURCHASE_AMOUNT = "구입하신 수동 로또의 개수가 전체 로또 개수보다 많습니다.\n관리자에게 문의하여 로또를 재구매하시기 바랍니다.";
 
     private List<Lotto> lottos;
 
-    public LottoMachine(Long purchaseAmount, LottoGeneratorStrategy lottoGeneratorStrategy) {
-        this.lottos = generateLottos(purchaseAmount, lottoGeneratorStrategy);
+    public LottoMachine(List<Lotto> manualLottos, int numberOfTotalLottos) {
+        this.lottos = generateLottos(manualLottos, numberOfTotalLottos);
     }
 
-    private List<Lotto> generateLottos(Long purchaseAmount, LottoGeneratorStrategy lottoGeneratorStrategy) {
-        int chances = validatePurchaseAmount(purchaseAmount);
-        return buy(chances, lottoGeneratorStrategy);
+    private List<Lotto> generateLottos(List<Lotto> manualLottos, int numberOfTotalLottos) {
+        validatePurchaseAmount(manualLottos, numberOfTotalLottos);
+        List<Lotto> lottos = setLottos(manualLottos, numberOfTotalLottos);
+        return lottos;
```

Comment:
변수 선언 후 사용되지 않는다면 바로 리턴해도 좋을 것 같아요
```suggestion
        return setLottos(manualLottos, numberOfTotalLottos);
```